cla_public/app.py

```diff
@@ -31,6 +32,85 @@
 
 def create_app(config_file=None):
     app = Flask(__name__)
+    # Adds security to Flask
+    csp = {
+        "default-src": ["'self'", "*.googletagmanager.com"],
+        "img-src": [
+            "'self'",
+            "*.googleapis.com",
+            "*.gstatic.com",
+            "*.google.com",
+            "*.googleusercontent.com",
+            "data:",
+            "*.googletagmanager.com",
+            "*.analytics.google.com",
+            "*.google.co.uk",
+            "*.g.doubleclick.net",
+            "*.google-analytics.com",
+        ],
+        "object-src": "'self'",
+        "script-src": [
+            "'unsafe-eval'",
```

Comment:
Would it be possible to leave a comment here explaining why `unsafe-eval` is enabled and that it should be removed when it is no longer required?

Reply:
agreed!

Reply:
Updated